pages/common/mogrify.md

```diff
@@ -0,0 +1,23 @@
+# Mogrify
```

Comment:
s/Mogrify/mogrify.

We always start a command with lower case.

Reply:
@agnivade, adding that check needs to be done in [tldr-lint](https://github.com/tldr-pages/tldr-lint), right?